tools/gmon/README.md

```diff
@@ -0,0 +1,141 @@
+# Cloud Monitoring CLI
```

Comment:
this readme could use an "installation" section.
e.g. how to get this added tool easily and permanently added to your path w/ necessary deps.

Reply:
added